ludwig/schema/encoders/text_encoders.py

```diff
@@ -60,6 +61,8 @@ class HFEncoderImplConfig(HFEncoderConfig):
         parameter_metadata=ENCODER_METADATA["HFEncoder"]["trainable"],
     )
 
+    tuner: Optional[BaseTunerConfig] = TunerDataclassField()
+
```

Comment:
If we're adding the `tuner` attribute to `HFEncoderImplConfig` and all other encoders are subclasses of this, maybe we just need it to add it here instead of adding it to every single encoder schema config?

Reply:
Sadly only `DebertaV2Config` inherits from `HFEncoderImplConfig` currently. For whatever reason these classes weren't made with an inheritance hierarchy. I added this stuff when I added Deberta so we could use this pattern going forward and refactor the existing stuff to use it, but unfortunately we haven't done that refactoring yet, so there's still a ton of duplication.